classes/ValidFormBuilder/Area.php

```diff
@@ -347,6 +353,21 @@ protected function __toHtml($submitted = false, $blnSimpleLayout = false, $blnLa
             $this->setMeta("class", "vf__clone");
         }
 
+        //*** If this is a removable field, add corresponding class
+        if ($this->isRemovable()) {
+            $this->setMeta("class", "vf__removable");
+        }
+
+        //*** Add data-dynamic="original" or data-dynamic="clone" attributes to dynamic fields
```

Comment:
Please explain in short why you add the data attributes. There is no direct logic to derive the "why" from.